utils/barista/commands/build/index.js

```diff
@@ -10,6 +30,13 @@ const chalk = require('chalk');
  */
 async function build (moduleType, config) {
   const { name } = config;
+  const newHash = await checkHash();
+
+  if (!newHash) {
+    console.log(chalk.green(`Hash matches previous build, skipping build for ${name}`))
```

Comment:
I'mma change this to `Skipping ${name}, source code is identical to previous build`
But i'd also say maybe we don't print anything at all? `CLI` doesn't need to be that verbose to the end user?

Reply:
yeah, let's not print anything